File: 1395-minimum-time-visiting-all-points/minimum-time-visiting-all-points.php
class Solution {

    /**
     * @param Integer[][] $points
     * @return Integer
     */
    function minTimeToVisitAllPoints($points) {
        $sm=0;
        for($i=0;$i<count($points)-1;$i++){
            $sm+=max(abs($points[$i+1][0]-$points[$i][0]),abs($points[$i+1][1]-$points[$i][1]));
        }
        return $sm;
    }
}

File: 1395-minimum-time-visiting-all-points/minimum-time-visiting-all-points.cs
public class Solution {
    public int MinTimeToVisitAllPoints(int[][] points) {
        int sum=0;
        for(int i=0;i<points.Length-1;i++){
            sum+=Math.Max(Math.Abs(points[i+1][0]-points[i][0]),Math.Abs(points[i+1][1]-points[i][1]));
                                        // System.out.println(sum);
        }
        return sum;
                                                            
    }
}